core/src/apps/cardano/layout.py

```diff
@@ -750,27 +750,56 @@ def _format_stake_credential(
         raise ValueError
 
 
-async def confirm_catalyst_registration(
+async def confirm_governance_registration_delegation(
     ctx: wire.Context,
     public_key: str,
-    staking_path: list[int],
-    reward_address: str,
-    nonce: int,
+    proportion: int,
 ) -> None:
+    props: list[PropertyType] = [
+        ("Governance voting key registration", None),
+        ("Delegating to:", public_key),
+    ]
+    if proportion is not None:
+        props.append(("Voting power proportion:", str(proportion)))
```

Comment:
I am not sure about this "proportion". CIP-36 and people on Slack talk about "weight", not "proportion". I've asked on Slack, see https://input-output-rnd.slack.com/archives/C036XSMFXE3/p1664970144958219

Reply:
As discussed on Slack, I renamed it to `weight` everywhere.